scripts/deps.py

```diff
@@ -31,9 +31,17 @@ def _setup(**kw): data.update(kw)
 if 'python' in spec: spec.remove('python')
 
 deps = ""
-deps += " ".join(s for s in spec)
+if sys.platform == "win32":
+    deps += '"'
+    deps += '" "'.join(s for s in spec)
+    deps += '"'
+else:
+    deps += '" "'.join(s for s in spec)
```

Comment:
this doesn't seem correct in the non-windows case. It looks like it would produce:
```
foo" "bar" "baz
```
with no quotes at the beginning. I think it needs to be `deps += " ".join(...)` as before. 

However, is it possible to a avoid changing this script at all, and put the necessary logic to handle this output in the windows scripts? AFAIK this works on windows with the original script:
```
for /F "delims=" %i in ('python scripts\deps.py build run') do (conda install %i)
```

Reply:
Tho perhaps that windows command is much slower (I think it does a separate install for each pkg?)

Reply:
On windows i never succeed to run the command of the documentation directly because of node <>= ineed to use quotes. With powershell or dos